App.js

```diff
@@ -21,6 +22,7 @@ export const UnconnectedApp = () => (
         <PermissionsProvider>
           <ExposureNotificationsProvider>
             <ExposureHistoryProvider>
+              <StatusBar barStyle={'dark-content'} />
```

Comment:
I'd put this top level / use the hook, so that the initialization of the connection steps doesn't gate the status bar

Reply:
makes sense to pull it out all together i think in that case as we are mostly overwriting the status style on a per screen basis.